assets/section-main-product.css

```diff
@@ -62,10 +62,18 @@
     width: calc(35% - var(--grid-desktop-horizontal-spacing) / 2);
   }
 
+  .product--medium:not(.product--no-media) .product__info-wrapper {
+    padding: 0 0 0 4rem;
+  }
+
   .product--large:not(.product--no-media).product--right .product__info-wrapper {
     padding: 0 4rem 0 0;
   }
 
+  .product--medium:not(.product--no-media).product--right .product__info-wrapper {
+    padding: 0 4rem 0 0;
+  }
+
```

Comment:
I wonder if those changes should only be applied when the color scheme are different 🤔 (`color scheme` vs `product information color scheme`).

Or if it's meant to be applied in all scenarios. 

I feel like it could always use 4rem above `750px` when set to medium and when the color scheme and product info color scheme don't match. Right now it's only above a `990px` viewport width.

Reply:
1. I prefer to keep it constant and apply to all scenarios no matter whether color-schemes are different or not.
2. I made it 4rem  only for a viewport wider than 990px on purpose. Olivia wanted it to be 5rem for Desktop media size set to Small. Even when the Desktop media size is set to Medium/Large if your screen is less than 990px the media size looks like a small one. That's why I decided to make it this way.

Reply:
Just to recap right now here is how the padding logic is working: 
- when `desktop media size` is set to large:
  - between 750px and 990px, padding value is `5rem`
  - above 990px, padding value is `4rem`
- when `desktop media size` is set to medium: 
  - between 750px and 990px, padding value is `5rem`
  - above 990px, padding value is `4rem`

On top of that when the product information is using a color scheme different than the rest of the section, you're losing another `4rem` of width for the text because of [the padding applied](https://screenshot.click/25-51-1mu5u-s6byw.png) which is `2rem` between `750px` and `990px` and `3rem` above `990px`.

I think there is a chance here for us to remove some lines of code, even if it's just a few, and leave that spacing at `4rem` at all time for whatever the media size is. 
Having a larger gap between the media column and product info on smaller screen doesn't make sense to me 🤔  

[Video to help visualize it](https://screenshot.click/25-01-dtiib-4ig4h.mp4)

Im going to cc: @Oliviammarcello to share context with her 🙂

Reply:
> I think there is a chance here for us to remove some lines of code, even if it's just a few, and leave that spacing at 4rem at all time for whatever the media size is.
Having a larger gap between the media column and product info on smaller screen doesn't make sense to me 

@ludoboludo Thanks for the detailed explanation, you convinced me! Let's make it 4rem regardless of media width.

Reply:
And should we do that only when the color scheme is different between the product info and product section 🤔 ? Since product info has some extra padding when it's different.

Reply:
I tried different padding sizes a lot and went to a conclusion that it looks the best for me when we have 3rem padding no matter what media size is but with different color schemes, and 4rem when the color schemes are same.

Reply:
@Oliviammarcello @ludoboludo @andrewetchen 

I made a video explaining my comment above. Also in the video I mentioned the question that Andrew brought up [here](https://github.com/Shopify/dawn/pull/1907#discussion_r955411467). And I forgot to mention in the video that the way I suggest looks more closer to Figma reference. 

Please let me know what you think. Thanks. 

Uploading PAdding size.mov…